srcCpp/RTIDDSImpl.cxx

```diff
@@ -112,6 +112,9 @@ void RTIDDSImpl<T>::Shutdown()
         if (!registry->unregister("wh", NULL, NULL)) {
             //printf("failed to unregister wh\n");
         }
+        if (!registry->unregister("_shmem", NULL, NULL)) {
```

Comment:
add a comment specifying that this will only be successful for Micro 3.0.0 and above.